Best Practices/Clean Code.md

```diff
@@ -1,88 +1,77 @@
 
 
 ### Naming
 
-Developers write code for machines to execute it, but for other developers to maintain and extend it. So code should be easy to read and understand.
-The code should reflect the shared vocabulary used by all team members involved in the project.
+Developers write code for machines to execute it, but for other developers to maintain it and extend it. So code should be easy to read and understand.
+Code should reflect shared vocabulary used by all team members involved in the project.
 
 So in general naming should be:
 
 * Well thought through, and should reflect business concepts and rules.
 * Consistent through the software.
-* Truthful - this applies especially on the method level, where the method name should not be too general or misleading due to the method's side effects.
+* Truthful - this applies especially on the method level, where method name should not be too general or misleading due to method's side effects.
 
 
 ### Comments
 
-Comments are part of the source code, and if not consisted of significant info, then comments act as noise. Even worse, if not well maintained they can lead developers to false conclusions. In general, developers should avoid writing comments. If the code is unclear, it is a sign it should be rewritten.
+Comments are part of source code, and if not consisted of significant info, then comments act as noise, and even worse if not well maintained they can lead developers to false conclusions. In general, developers should avoid writing the comments. If the code is unclear, it is a sign it should be rewritten.
 
 Exceptions:
 
 * Describing an example
-* Pointing to resources in the documentation
+* Pointing to resources in documentation
 * Todos
 * Swagger documentation
 
 
 ### Methods
 
-Methods should be short and have a single responsibility.
-The logic contained in a single method should reflect the same level of abstraction. Mixing different levels of abstraction in the same function is a code smell.
+Methods should be short and have single responsibility.
+Logic contained in a single method should reflect the same level of abstraction. Mixing different levels of abstraction in the same function is a code smell.
 
 * Methods should not have side effects, and method names should reflect exactly what they are doing.
 * Prefer methods no longer than 10 lines.
-* The number of input parameters should be up to 4. If there is a need for more parameters, consider creating a DTO.
+* Number of input parameters should be up to 4. If there is a need for more params, consider creating a DTO.
 * Methods should not have multiple return params (exception is TryDoSomething pattern which returns bool and resulting object via out param).
-* Avoid using flag arguments. Split the method into several independent methods that can be called from the client without the flag.
+* Avoid using flag arguments. Split method into several independent methods that can be called from the client without the flag.
```

Comment:
I wanted to target the whole document, but couldn't, so apply this comment to everything above the changes you added - can you please double-check all the changes here? It seems like you might have branched out on an older version of the doc, I remember working on these changes, but in the other direction :)

Reply:
I checked line by line on master and staging, it seems like these changes you pointed out aren't there... I know that I didn't add them, I only added the SOLID principles :D But since they aren't present on staging and typos are removed with them, should we keep the changes anyway?